src/snowflake/snowpark/session.py

```diff
@@ -892,6 +892,11 @@ def table(self, name: Union[str, Iterable[str]]) -> Table:
             name: A string or list of strings that specify the table name or
                 fully-qualified object identifier (database name, schema name, and table name).
 
+            Note:
+                If your table name has special characters, please use double quotes to mark it like ``session.table('"my table"')``.
```

Comment:
Optionally, I suggest changing "has" to "contains". 
Remove "please".
Add "this," after "mark it like".